apps/docs/src/app/contact/page.tsx

```diff
@@ -1,11 +1,19 @@
 import { Heading, Paragraph } from '@govie-react/ds';
+import { config } from '@/lib/config';
 
 export default function ContactPage() {
   return (
     <div>
       <Heading>Contact</Heading>
       <Paragraph>
-        You can contact the design system team via email. Email coming soon.
+        You can contact the design system team using{' '}
+        <a
```

Comment:
nitpick: You can use `<Link>` component from `govie-react/ds`.

Reply:
Good idea. Though if we use the `Link` component from `govie-react/ds` we won't be able to open the form in a new tab as it doesn't have a `target` prop yet

Reply:
maybe we should add a TODO as Ian was suggesting